data/EdlienTheory.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "Edlien Theory",
+  "type": "community",
+  "bio": "I've brought you a bargain community member, become a open source legend in trade of never using non-inclusive language",
+  "tags": [
+     "Theory",
+     "Eddie Jaoude",
+     "EddieHubCommunity",
+     "OpenSource"
+  ],
+  "links": [
+    {
+      "group": "Theory",
+      "name": "Edlien is a what we call the true form of Eddie",
+      "url": "#",
```

Comment:
What are these links going to be used for? 🤔

Reply:
@krshkun can explain this better than anyone. It should be okay for approval.

Reply:
This will lead users to `This page can't be reached` or something. 

One thing we could do is to have a repo in his org containing all the theories, then we'll have valid links for these theories + people will able to contribute more theories! Wdyt @krshkun!? 🤷🏻‍♂️

Reply:
Nice Idea! I'll add a repo and a link to the repo